### PR TITLE
Adds description to AudioWorklet basic demo pages

### PR DESCRIPTION
**Pull request by hoch**
Fixes: #121.

- Adds descriptions to AudioWorklet basic demo pages.
- Updates the demo style.
- Now the demo helpers uses a button to start the demo.

### CODE REVIEW

File: audio-worklet/basic/bit-crusher.html [hunk 1]
```diff
@@ -20,25 +33,28 @@
         const context = new AudioContext();
         let oscillator = new OscillatorNode(context);
         let bitCrusher = new AudioWorkletNode(context, 'bit-crusher');
+
+        oscillator.type = 'sawtooth';
+        oscillator.frequency.value = 5000;
```
Comment (rtoy) [inline audio-worklet/basic/bit-crusher.html]: Why the change in oscillator type?  Better sounding?

File: audio-worklet/basic/bit-crusher.html [hunk 2]
```diff
@@ -20,25 +33,28 @@
...
         paramBitDepth.setValueAtTime(1, 0);
 
-        // The target reduction values are determinded experimentally.
+        // The target reduction values are determined experimentally.
         paramReduction.setValueAtTime(0.01, 0);
         paramReduction.linearRampToValueAtTime(0.1, 4);
         paramReduction.exponentialRampToValueAtTime(0.01, 8);
 
-        // Play the tone for 8 seconds.
+        // Play the tone for 9 seconds.
```
Comment (rtoy) [inline audio-worklet/basic/bit-crusher.html]: 8, right? stop(1) would be 1 sec.

File: audio-worklet/basic/bit-crusher.html [hunk 3]
```diff
@@ -2,16 +2,29 @@
...
-    <h1>
-      Demo: BitCrusher
-    </h1>
+    <section>
+      <h1>
+        Demo: BitCrusher with AudioParam Automation
+      </h1>
+      <dt>A BitCrusher example from the specification, but modified to
+      demonstrate AudioParam automations. The demo runs for 8 seconds.</dt>
+      <dd>
```
Comment (rtoy) [inline audio-worklet/basic/bit-crusher.html]: Maybe say the source is an oscillator and say the kinds of automation if there are only a couple.

File: audio-worklet/basic/hello-audio-worklet.html [hunk 4]
```diff
@@ -9,9 +9,21 @@
...
-    <h1>
-      Demo: "Hello AudioWorklet!"
-    </h1>
+    <section>
+      <h1>
+        Demo: "Hello AudioWorklet!"
+      </h1>
+      <dt>A simple AudioWorkletNode that bypasses the incoming audio stream to
+      its output.</dt>
+      <dd>
```
Comment (rtoy) [inline audio-worklet/basic/hello-audio-worklet.html]: Maybe say what the source is.

File: audio-worklet/basic/noise-audio-param.html [hunk 5]
```diff
@@ -2,16 +2,30 @@
...
-    <h1>
-      Demo: AudioParam Modulation
-    </h1>
+    <section>
+      <h1>
+        Demo: Noise with AudioParam Modulation
+      </h1>
+      <dt>
+        A simple noise generator with user-defined AudioParam modulated by an
+        OscillaltorNode.
```
Comment (rtoy) [inline audio-worklet/basic/noise-audio-param.html]: Typo.

File: audio-worklet/basic/noise-audio-param.html [hunk 6]
```diff
@@ -2,16 +2,30 @@
...
     <a href="../index.html" class="link-small">&#171; AudioWorklet home</a>
-    <h1>
-      Demo: AudioParam Modulation
-    </h1>
+    <section>
+      <h1>
+        Demo: Noise with AudioParam Modulation
+      </h1>
+      <dt>
+        A simple noise generator with user-defined AudioParam modulated by an
```
Comment (rtoy) [inline audio-worklet/basic/noise-audio-param.html]: Explain briefly what the audio param is modulating.

File: audio-worklet/basic/one-pole.html [hunk 7]
```diff
@@ -9,9 +9,21 @@
...
   <body>
     <a href="../index.html" class="link-small">&#171; AudioWorklet home</a>
-    <h1>
-      Demo: "One Pole Filter"
-    </h1>
+    <section>
+      <h1>
+        Demo: "One Pole Filter"
+      </h1>
+      <dt>An <a href="https://ccrma.stanford.edu/~jos/fp/One_Pole.html">one-pole
```
Comment (rtoy) [inline audio-worklet/basic/one-pole.html]: Typo: "An" -> "A"

File: audio-worklet/basic/one-pole.html [hunk 8]
```diff
@@ -9,9 +9,21 @@
...
+      </a>filter implementation with AudioWorkletNode.</dt>
```
Comment (rtoy) [inline audio-worklet/basic/one-pole.html]: Explain what the source is, and what's happening to the filter.  Otherwise, you don't know what the output should sound like.

File: audio-worklet/index.html [hunk 9]
```diff
@@ -8,31 +8,46 @@
...
+        </li>
+      </ul>
+      <dt>AudioWorklet is currently available on Chrome behind the
+      experimental flag.<br>Use the following
+      <a href="https://www.chromium.org/developers/how-tos/run-chromium-with-flags">
+      command line option</a> to activate the feature.</dt>
+      <dd><code class="flag">--enable-blink-features=Worklet,AudioWorklet</code>
+      </dd>
+      <dt>Or you can enable Experimental Web Platform Features in Chrome's flag
+      setting.<br>Note that this setting enables all the experimental features
```
Comment (rtoy) [inline audio-worklet/index.html]: Not sure about the style, but shouldn't these be in `<p>` tags, and no `<br>`?

File: index.html [hunk 10]
```diff
@@ -15,219 +15,223 @@
...
-        <a href="samples/audio/wavetable-synth.html">wavetable-synth.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/wavetable-synth2.html">wavetable-synth2.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/zero-crossings.html">zero-crossings.html</a>
-      </li>
-    </ul>
+    <section>
```
Comment (rtoy) [inline index.html]: What's happening here? Hard to tell from the diff.

File: resources/base.css [hunk 11]
```diff
@@ -14,30 +14,64 @@ a:hover {
 }
 
 a.link-small {
-  font-size: 12px;
+  font-size: 13px;
```
Comment (rtoy) [inline resources/base.css]: Why the change?

File: resources/base.css [hunk 12]
```diff
@@ -14,30 +14,64 @@ a:hover {
...
   line-height: 26px;
+  padding: 0;
+}
+
+h1 {
+  font-size: 34px;
+}
+
+h2 {
+  font-size: 28px;
```
Comment (rtoy) [inline resources/base.css]: Are these really necessary?  Are the defaults wrong in some way?

File: resources/base.css [hunk 13]
```diff
@@ -14,30 +14,64 @@ a:hover {
...
+}
+
+dd {
+  margin-top: 21px;
+}
+
+code.flag {
+  font-size: 16px;
+}
+
```
Comment (rtoy) [inline resources/base.css]: Are the defaults wrong in some way?

File: audio-worklet/basic/bit-crusher.html (hunk 1 above)
Comment (hoch) [reply audio-worklet/basic/bit-crusher.html]: Yes, it really shows off the nice effect of bit crusher. More higher frequency content, more crushing sounds.

File: audio-worklet/basic/bit-crusher.html (hunk 3 above)
Comment (hoch) [reply audio-worklet/basic/bit-crusher.html]: The kinds of automation can be found with a single click 1 inch below. :)
I'll clarify the source.

File: index.html (hunk 10 above)
Comment (hoch) [reply index.html]: I've added `<section>` elements. So the indentation is changed.

File: resources/base.css (hunk 11 above)
Comment (hoch) [reply resources/base.css]: I thought the small link was too small.

File: resources/base.css (hunk 12 above)
Comment (hoch) [reply resources/base.css]: We're not using default any way. I am making incremental changes on the visual structure of this demo depository in the future too. The style should be minimal, but also should be modern enough from my POV.

**Comment by rtoy** [comment]
Got a preview link?

**Comment by hoch** [comment]
Here: [Preview](https://cdn.rawgit.com/GoogleChromeLabs/web-audio-samples/121-add-description-to-demos/index.html)

**Comment by hoch** [comment]
Oh, it's not a custom font. It's "monospace", which is default for `<code>`.
